pkg/plugin/istio/install.go

```diff
@@ -147,6 +147,13 @@ func (p *IstioPlugin) createIstioCacerts() error {
 	if err == nil {
 		// skip create cacerts if exists
 		logrus.Infof("secret %s already exists, skipping create", caSecret)
+		if len(s.TypeMeta.Kind) == 0 {
+			logrus.Debugf("s.len(s.TypeMeta.Kind) == 0")
+			s.TypeMeta = metav1.TypeMeta{
+				APIVersion: "v1",
+				Kind:       "Secret",
+			}
+		}
```

Comment:
could not understand why we need to explicitly add this meta, doesn't client-go return complete Secret?

Reply:
yes, and don't know why.
![1](https://user-images.githubusercontent.com/45359033/196899079-e8dcccf1-3d28-4a58-bea2-11d62a1cf72b.PNG)

Reply:
This is not clear, which object is missing type meta, can you add more debug logs

Reply:
In theory it is not possible to get secret from kube-apiserver miss type meta, I doubt it is some other resource that constructed by our code

Reply:
Found this issue https://github.com/kubernetes/client-go/issues/413

Reply:
nit: use the typemeta

Reply:
done